sbt-metals/src/main/scala/MetalsPlugin.scala

```diff
@@ -1,91 +1,91 @@
+package scala.meta.sbt
+
 import sbt._
 import sbt.Keys._
 import java.io._
-import sbt.Def
 
 object MetalsPlugin extends AutoPlugin {
```

Comment:
Can we move this file to `scala/meta/sbt/MetalsPlugin.scala`? Or else IntelliJ complains about a wrong location. I think it's a good practice to match directory with package.